src/goose/cli/session.py

```diff
@@ -90,11 +92,13 @@ def __init__(
         name: Optional[str] = None,
         profile: Optional[str] = None,
         plan: Optional[dict] = None,
+        tracer: trace.Tracer = trace.get_tracer(__name__),
```

Comment:
I think this names the tracer, aka instrumentation scope, `goose.cli.session`. I would just use `goose` or `square.goose`

Also I think it's conventional to make `get_tracer` a module variable rather than a parameter, customization is almost always in the sdk side, not API. i.e., unit tests mock the exporter, not the API

Reply:
I had missed that tests are run in parallel

https://github.com/square/goose/pull/75/files#diff-412a40a9e31259c93343c77f7859015343e40b535b6a9f55062b15b6ab23ad3aR21

The module-level approach probably won't work well with that so it seems fine to leave as is, sorry for the noise.

Reply:
added missing comments, thanks!